src/components/DataSpecSelector/DataSpecSelector.js

```diff
@@ -49,6 +49,16 @@ export default class DataSpecSelector extends React.Component {
       _.sortBy(ids, (item) => item[1]),
       (item) => item[1],
     );
+    ids = _.sortedUniqBy(
+      _.sortBy(ids, (item) => item[1]),
+      (item) => item[1],
+    );
+    if (ids[0][1].split(" ")[0] == ids[ids.length - 1][1].split(" ")[0]) {
```

Comment:
Never use `==` comparison; its type-coercion is too tricky and leads to errors. Use only strict equality `===`. This is standard advice in JS land.

Reply:
Also, it's hard to deduce what is being compared here and why. Suggest you decompose the `ids` items into helpfully-named values that you then compare. Something like

```
const first_run_id = ids[0][1].split(" ")[0];
const last_run_id = ids[ids.length - 1][1].split(" ")[0];
if (first_run_id === last_run_id) {
   ...
```

If you were feeling really fun you could define a temp function `const run_id = id = id[1].split(" ")[0]` to DRY things up.

Reply:
Alternatively, could the condition be evaluated by 

```
import { hasMultiRuns } from "../../map-controllers/map-helpers";
```

as it is in another component further down?